modules/deployment/src/test/resources/META-INF/mule-module.properties

```diff
@@ -1,3 +1,6 @@
-    module.name=deployment-test
+module.name=deployment-test
 
-artifact.export.classPackages=org.mule.runtime.module.deployment.internal
+artifact.export.classPackages=org.mule.runtime.module.deployment.internal,\
+                              org.mule.functional.policy
```

Comment:
the way the deployment tests run in a non-isolated manner, so the config from the plugin is not used. Anyway, i don't need to export this package here, but i still need to export the xsd.